src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.CurlResponseMessage.cs

```diff
@@ -302,8 +302,8 @@ public override Task<int> ReadAsync(byte[] buffer, int offset, int count, Cancel
                     // in the buffer so we need to register ourself to get the next write.
                     if (cancellationToken.CanBeCanceled)
                     {
-                        // If the cancellation token is cancelable, then we need to register for cancellation.
-                        // We creat a special CancelableReadState that carries with it additional info:
+                        // If the cancellation token is cancellable, then we need to register for cancellation.
```

Comment:
cancelable is a fine variation.  Let's revert these.  We've tried to use cancel, canceling, canceler, cancelable, etc., with the only two-l variation being cancellation.